api/v1/aerospikecluster_types.go

```diff
@@ -756,7 +756,8 @@ type AerospikeClusterStatus struct { //nolint:govet // for readability
 	Pods map[string]AerospikePodStatus `json:"pods" patchStrategy:"strategic"`
 
 	// Phase denotes the current phase of Aerospike cluster operation.
-	Phase AerospikeClusterPhase `json:"phase,omitempty"`
+	Phase    AerospikeClusterPhase `json:"phase,omitempty"`
+	Selector string                `json:"selector"`
```

Comment:
Why selector is not omitempty? Will it not cause the issue during the upgrade?

Reply:
Yeah, it'll fail in CR validation against new CRDs during upgrade.
Making it omitempty